cirrus/server/cirrus/main.py

```diff
@@ -109,6 +110,41 @@ def initialize_glean():
     return pings, metrics
 
 
+def get_metrics_data(
+    enrolled_partial_configuration: Dict[str, Any]
+) -> List[tuple[str, str, str, str]]:
+    events: List[Dict[str, Any]] = enrolled_partial_configuration.get("events", [])
+    data: List[tuple[str, str, str, str]] = []
+    for event in events:
+        if event.get("change") == "Enrollment":
+            experiment_slug: str = event.get("experiment_slug", "")
+            branch_slug: str = event.get("branch_slug", "")
+            enrollment_id: str = event.get("enrollment_id", "")
```

Comment:
Please do take note of the current and ongoing discussion around the enrollment id here: https://bugzilla.mozilla.org/show_bug.cgi?id=1838059

Unless the enrollment id is absolutely necessary, I would recommend omitting it.